### DISCUSSION TITLE
Any Discord for the community ?

### DISCUSSION THREAD

**Discussion by MatthieuHernandez** [hsutter/cppfront#1300] [Suggestions]
Hello, I'm new here, having just started a cppfront project this week.
As a developer and gamer, I use Discord daily. Is there a cppfront Discord?
I think this would be a great help in shaping the community. Discord would be a great tool for chatting, exchanging language, sharing projects, asking questions, asking for help, making surveys, and so on.

If there isn't one but other people are interested, I'd be happy to help create one.

**Comment by MaxSagebaum** [comment]
Currently the community resolves around the github repo. Here you can ask questions, open new threads etc. The only thing missing is the ability to start a vocal chat. 

The problem I always have with discord is that you can not start your own topics. You just have general ones. Keeping an overview of the different discussions in one general topic is quite hard.

Nevertheless, a small discord which lovers the barrier of asking small and simple questions could help. For larger topics/discussions the people could be refereed to the github page.

**Comment by MatthieuHernandez** [comment]
With Discord, it is possible to start topics and to have discussion threads with the ‘forums’ feature. But yes, I agree with you, Discord is more suitable for naive and small questions and Github is better for large one or more community-based subjects.

_Example of a forum:_
![image](https://github.com/user-attachments/assets/6f3f5d73-b065-45c4-ae7e-626a593789bc)
